src/components/content/catalog/services/details/ServiceProvider.tsx

```diff
@@ -208,47 +205,129 @@ function ServiceProvider({
-                                    serviceRegistrationStatus={
-                                        activeServiceDetail.serviceTemplateRegistrationState as serviceTemplateRegistrationState
-                                    }
+                                    availableInCatalog={activeServiceDetail.availableInCatalog}
                                 />
                                 <ReRegisterService
-                                    id={activeServiceDetail.serviceTemplateId}
                                     setIsViewDisabled={setIsViewDisabled}
                                     reRegisterRequest={reRegisterRequest}
-                                    serviceRegistrationStatus={
-                                        activeServiceDetail.serviceTemplateRegistrationState as serviceTemplateRegistrationState
-                                    }
+                                    activeServiceDetail={activeServiceDetail}
                                 />
                                 <DeleteService
                                     id={activeServiceDetail.serviceTemplateId}
                                     setIsViewDisabled={setIsViewDisabled}
-                                    serviceRegistrationStatus={
-                                        activeServiceDetail.serviceTemplateRegistrationState as serviceTemplateRegistrationState
-                                    }
+                                    activeServiceDetail={activeServiceDetail}
                                 />
                             </div>
-                            <h3 className={catalogStyles.catalogDetailsH3}>
-                                <EnvironmentOutlined />
-                                &nbsp;Service Hosting Options
-                            </h3>
-                            <ServiceHostingOptions
-                                serviceTemplateDetailVos={groupServiceTemplatesByCsp.get(serviceCspInQuery) ?? []}
-                                defaultDisplayedService={activeServiceDetail}
-                                serviceHostingTypeInQuery={serviceHostingTypeInQuery}
-                                updateServiceHostingType={onChangeServiceHostingType}
-                            />
-                            <ServiceDetail serviceDetails={activeServiceDetail} />
-                            <ServicePolicies
-                                key={activeServiceDetail.serviceTemplateId}
-                                serviceDetails={activeServiceDetail}
-                                isViewDisabled={isViewDisabled}
-                            />
+                            {(activeServiceDetail.serviceTemplateRegistrationState === 'in-review' ||
+                                activeServiceDetail.serviceTemplateRegistrationState === 'approved') &&
+                            !activeServiceDetail.availableInCatalog ? (
+                                <Badge.Ribbon
```

Comment:
still the display is wrong when update review is in progress and the original template is still available in catalog. I see only 'review in progress'.

And more over, re-register button is enabled in this case. Which is also wrong. 

![image](https://github.com/user-attachments/assets/b56c110e-5704-4c39-a384-0cb66c7c2607)